examples/bert/run_glue_finetuning.py

```diff
@@ -200,13 +213,15 @@ def preprocess_data(inputs, labels):
 
     # Read and preprocess GLUE task data.
     train_ds, test_ds, validation_ds = load_data(FLAGS.task_name)
-    train_ds = train_ds.batch(FLAGS.batch_size).map(
+
+    # batch_size is taken as 32.
+    train_ds = train_ds.batch(32).map(
```

Comment:
I would still keep these in a constant at least at the top of the file.

Reply:
We discussed about this [here](https://github.com/keras-team/keras-nlp/issues/114#issuecomment-1109204202). So, I made the `epochs` and `batch_size` as constants with values `3` and `32` as mentioned in the paper.

Reply:
thanks!

3 and 32 are good default value, but I would prefer to keep `epochs` and `batch_size` as flags, so it is more clear for people to read.

Reply:
Yup, I've changed it in the latest commits.